index.js

```diff
@@ -364,15 +364,15 @@ BabelClient.prototype.createAnnotation = function createAnnotation(token, data,
         if(err){
             callback(err);
         } else{
-            if(body.message && body.errors){
+            if(!this.responseSuccessful(response) || (body.message && body.errors)){
```

Comment:
I thought we determined it was returning `.error` as part of the `body` and not `.errors`?

Reply:
I've not changed what the existing code is doing here re error/errors - I've just added a check for the status code.

The client is checking for 'errors' - but at least some responses were returning 'error' as you say. This must always have been broken. But that approach is flawed - as evidenced by this bug. I don't think we should change the error/errors checking, because I'm not sure of all the possible responses from babel. But I am sure that if we return a status code of 400, it should not be a successful request.